pyproject.toml

```diff
@@ -108,10 +109,15 @@ dev = [
 [tool.hatch.build.targets.wheel]
 packages = ["src/antsibull_docs", "src/sphinx_antsibull_ext"]
 
+[tool.black]
+exclude = '''
+tests/functional/baseline-sphinx-init-.*
+'''
+
 [tool.isort]
 # Match black
 profile = "black"
-line_length = 89
+line_length = 88
```

Comment:
Without this, isort reformats a line that black instantly changes back.

Reply:
Hmm, I should have realized that before I copied it to all the antsibull projects... Thanks for catching it :).

Looking at https://pycqa.github.io/isort/docs/configuration/profiles.html, I think setting `profile = "black"` and removing the other options is a better choice. The black profile includes everything else that is set here.

Reply:
Ah, I didn't knew that 88 was the 'right' value. I simply decreased 89 by one and then the modify-and-back cycle didn't happen anymore...